base/deprecated.jl

```diff
@@ -292,36 +292,7 @@ export call
 
 #15409
 # Deprecated definition of pmap with keyword arguments.
-# When this is removed the following definition needs to be uncommented
-# and added to pmap.jl
-# pmap(f, c...) = pmap(default_worker_pool(), f, c...)
-
-function pmap(f, c...; err_retry=nothing, err_stop=nothing, pids=nothing, kwargs...)
-    kwargs = Dict{Symbol, Any}(kwargs)
-
-    if err_retry != nothing
-        depwarn("err_retry is deprecated, use pmap(retry(f), c...).", :pmap)
-        if err_retry == true
-            f = retry(f)
-        end
-    end
-
-    if pids == nothing
-        p = default_worker_pool()
-    else
-        depwarn("pids is deprecated, use pmap(::WorkerPool, f, c...).", :pmap)
-        p = WorkerPool(pids)
-    end
-
-    if err_stop != nothing
-        depwarn("err_stop is deprecated, use pmap(f, c...; on_error = error_handling_func).", :pmap)
-        if err_stop == false
-            kwargs[:on_error] = e->e
-        end
-    end
-
-    pmap(p, f, c...; kwargs...)
-end
+# deprecation warnings are in pmap.jl
```

Comment:
Well, I tried. Unfortunately we simply don't have a way to deprecate a subset of keyword arguments using a separate definition.